static/front-style.css

```diff
@@ -18,8 +18,60 @@
   background-color: #4caf50;
   color: white;
   cursor: pointer;
+/* Responsiveness for desktop */
+/* Containers */
+.smaily-layout-1-container, .smaily-layout-2-container {
+	display: flex !important;	
+}
+.smaily-layout-3-container {
+	margin: 0 0.4rem 0 0 !important;
+}
+.smaily-layout-4-container {
+	display: flex !important;
+	max-width: 700px;
+}
+
+/* Inputs */
+.smaily-layout-1-input, .smaily-layout-4-input-email, .smaily-layout-4-input-name  {
+	margin: 0 0.8rem 0 0 !important;
+}
+.smaily-layout-3-input {
+	margin: 0 0 1.8rem 0 !important;
+}
+.smaily-layout-5-input {
+	margin: 0 0 0.8rem 0 !important;
 }
 
 .smaily-newsletter-form button[type="submit"]:hover {
   background-color: #45a049;
+/* Smaily button */
+.smaily-button {
+	white-space: nowrap;
+}
+
+/* Responsiveness for mobile */
+@media screen and (max-width: 450px) {
+	/* Containers */
+	.smaily-layout-1-container {
+		display: flex !important;
+	}
+	.smaily-layout-2-container {
+		display: flex !important;
+	}
```

Comment:
These rules can be omitted, because they are exactly the same as rules defined on line 23 - `.smaily-layout-1-container, .smaily-layout-2-container`.